Fix default fallback of IMU_GYRO_FFT_LEN param

PedroHRPBS-tii opened the pull request:
## Describe problem solved by this pull request
Setting IMU_GYRO_FFT_LEN to any values that would make it fall under the "default" case at GyroFFT::init(), would cause a null pointer exception resulting in the board getting stuck on boot.

## Describe your solution
The default case was updating the parameter, but not updating the _rfft_q15 attributes, I simply added the lines that would refer to the default case of IMU_GYRO_FFT_LEN=256

## Test data / coverage
It was tested by defining values to IMU_GYRO_FFT_LEN that would fall in the default case, such as 0 or 1. If those values are defined on the SDCard config.txt, the fallback to default 256 is handled properly now.

Commit:
Fix default fallback of IMU_GYRO_FFT_LEN param

dagar commented:
Thanks @PedroHRPBS-tii. How about simply rearranging the case ordering so default: simply falls through to case 256?

PedroHRPBS-tii commented:
@dagar thanks for the feedback! I rearranged the ordering as you suggested. Also I removed the specific case of 256, as it's already the default. Finally, I added a warning in case it's set to something different than 256, so the end-user doesn't get confused on why it's not accepting different values.

I'm not sure why there are some cases that are commented out, I'd say they should be deleted, but I don't know what was the thought process behind it (I didn't touch them).

Let me know your thoughts on that.

dagar commented:
> I'm not sure why there are some cases that are commented out, I'd say they should be deleted, but I don't know what was the thought process behind it (I didn't touch them).

The entire thing is kind of an experiment that's likely going to be replaced with a sliding DFT that updates synchronized with the inner loop.
